### PR TITLE
Airframe and mixer for rover DF Robot GPX:Asurada

### PR DESCRIPTION
**Pull request by KatrinMoritz**
**Describe problem solved by this pull request**
Added a new airframe for the DF Robot GPX:Asurada rover. This rover has 2 motors and one servo.

**Describe your solution**
Added the airframe _50004_dfrobot_gpx_asurada_ and the mixer _rover_diff_and_servo.main.mix_. The airframe is based on the Generic Ground Vehicle airframe. The default PWM values for output 3 and 4 are changed because the included ESCs have other default values.
The mixer file was designed as follows: The servo controls the front wheels via PWM Output 2. The 2 motors control each rear wheel. The PWM Output 3 controls the right rear wheel and PWM Output 4 controls the left rear wheel. A light differential is integrated in the motors. So that the outer wheel runs faster than the inner wheel during steering.

**Describe possible alternatives**
The mixer also can be configured as an Multirotor mixer for including the differential. But then the Outputs have to be changed. Then steering should be connected to PWM Channel 5 and the motors should be connected to PWM Channel 2 and 4.

**Test data / coverage**
Rover was tested in manual mode. This worked as expected.

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/airframes/50004_dfrobot_gpx_asurada [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	param set GND_SPEED_IMAX 0.125
+	param set GND_SPEED_THR_SC 1
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
+
+	param set NAV_ACC_RAD 0.5
+
+	# Provide ESC a constant 1500 us pulse, which corresponds to
+	# idle on the Roboclaw motor controller on the Aion R1
```
Comment (julianoes) [inline ROMFS/px4fmu_common/init.d/airframes/50004_dfrobot_gpx_asurada]: I'm confused how this one is related to the Roboclaw motor controller. Is this comment required?

Comment (KatrinMoritz) [reply ROMFS/px4fmu_common/init.d/airframes/50004_dfrobot_gpx_asurada]: No, I think I have forgot to delete/change this comment.

**Comment by LorenzMeier** [comment]
Should be squashed when merging.

**Comment by LorenzMeier** [comment]
CI failure is a false positive. I've filed this issue for the CI bug: https://github.com/PX4/Firmware/issues/14324

**Comment by igalloway** [comment]
@LorenzMeier Question - I'm thinking people will have difficulty finding this frame as it was custom built for NXP-CUP cars. Would it be appropriate to name this to include "NXP-CUP" in the name or is the rule of thumb manufacture only? (Our intent is to provide an NXP-CUP kit with this hardware directly from NXP.)

**Pull request merged**